source/includes/_endpoint-dataset.md

```diff
@@ -130,6 +139,170 @@ is_published | boolean | true | Indicates if the dataset is published to viewers
+                                        "blue"
+                                    ],
+                                    "dataset": "https://beta.crunch.io/api/datasets/173b4eec13f542588b9b0a9cbcd764c9/",
+                                    "dataset_id": "173b4eec13f542588b9b0a9cbcd764c9",
+                                    "dataset_created_time": null,
+                                    "subvar_names": [],
+                                    "dataset_name": "econ_few_columns_0",
+                                    "description": "In what year were you born?"
+                                }
+                            }
+                        },
+                        "description": ""
+                    },
+                    "https://beta.crunch.io/api/datasets/4473ab4ee84b40b2a7cd5cab4548d584/": {
+                        "labels": [],
+                        "name": "simple_alltypes",
+                        "description": ""
+                    }
+                },
+                "variables": {
+                    "https://beta.crunch.io/api/datasets/4473ab4ee84b40b2a7cd5cab4548d584/variables/000000/": {
+                        "dataset_labels": [],
+                        "users": [
+                            "00002"
+                        ],
+                        "alias": "x",
+                        "dataset_end_date": null,
+                        "category_names": [
+                            "red",
+                            "green",
+                            "blue",
+                            "4",
+                            "8",
+                            "9",
+                            "No Data"
+                        ],
+                        "dataset_start_date": null,
+                        "name": "x",
+                        "dataset_description": "",
+                        "dataset_archived": false,
+                        "group_names": null,
+                        "dataset": "https://beta.crunch.io/api/datasets/4473ab4ee84b40b2a7cd5cab4548d584/",
+                        "dataset_id": "bb987b45a5b04caba10dec4dad7b37a8",
+                        "dataset_created_time": null,
+                        "subvar_names": [],
+                        "dataset_name": "export test 94",
+                        "description": "Numeric variable with value labels"
+                    }
+                },
+                "variable_count": 14,
+                "totals": {
+                    "variables": 4,
+                    "datasets": 2
+                }
+            }
+        ]
+    }
+}
+```
+
+The variables grouping displays metadata for all of the variables that matched.
+The Datasets grouping displays metadata for all of the datasets where a variable or the dataset it self matched.  The "groups"
+parameter of the dataset indicates any groups that matched, along with variables that did not match the search but are contained
```

Comment:
What are "groups" in this context? Elements of the variables' hierarchical order? Perhaps link to that page in the docs, to be clear. Or be more explicit that search is searching order group names.

Come to think of it, it might be good to document what metadata we are searching in.